code/modules/ghostroles/spawner/human/responseteams/human/biesel.dm

```diff
@@ -0,0 +1,38 @@
+/datum/ghostspawner/human/ert/tcaf
```

Comment:
Only the leadership role in a species restricted ERT may be whitelisted.
All the other roles must be accessible without a whitelist.

Reply:
The roles are all accessible without a whitelist for the TCAF one as they're all playable as humans - this was taken from the TCFL one which also uses whitelists for the individual species